bust-a-move-framework/src/main/java/bustamove/App.java

```diff
@@ -24,88 +26,115 @@
  */
 
 public class App extends StateBasedGame {
-    /**
-     * Dimensions of the game window.
-     */
-    private static final int GAME_WIDTH = 640;
-    private static final int GAME_HEIGHT = 580;
-    /**
-     * Values for fps of the game.
-     */
-    private static final int MAX_FPS = 120;
+	/**
+	 * Dimensions of the game window.
+	 */
+	public static final int GAME_WIDTH = 640;
+	public static final int GAME_WIDTH_MULTIPLAYER = 1040;
+	public static final int GAME_HEIGHT = 580;
```

Comment:
Also fix the longer than 80 chars violation. If you can't run the project as a maven build, then install the checkstyle plugin for eclipse and use that instead.